automations-to-level-up-your-blog-with-directus/index.md

```diff
@@ -0,0 +1,215 @@
+Now add a **Read Data** operation attached to the Content Translations collection with the ID `{{$trigger.keys[0]}}`.  This will return the payload that includes the `title` and `detail` of the content that was just updated. If you look at the payload, this data does not come with the value for `approved`, this is because it is saved in the `content` collection. So we need to use the **Read Data** operation again, but this time we will attach to the `content` collection with the ID `{{$trigger.keys[0]}}`.
+Now use the Run Script operation to run the following JavaScript code:
+```javascript
+module.exports = async function(data) {
+    // Initialize an empty result object
+    let result = {};
+
+    // Check if "approved" is "yes"
+    if (data.item_read_rif6q.approved === "yes") {
+        // Publish the article title and detail
+        result = {
+            title: data.item_read_6grfu.title,
+            detail: data.item_read_6grfu.detail
+        };
+    } else {
+        result = {
+            status: "Not Approved for Publishing"
+        };
+    }
+    
+    return result;
+};
+```
+
+In the code above, `item_read_rif6q` and `item_read_6grfu` are the keys for the Read Data operations, so yours might be different since they are automatically generated.
+This is what your output will look like this when a content is approved:
+```json
+{
+    "title": "Reasons why automating delivery could be a bad idea ",
+    "detail": "Lack of Human Oversight:\nAutomated systems can lack the nuanced judgment that humans provide. Critical decisions might require human intervention to account for context, unexpected situations, or complex problem-solving...."
+}
+```
+## 3. Configuring Alerts for New Comments
+![comment alert](comment-alert.png)
+
+As your blog grows, monitoring comments manually becomes impractical. Automated alerts ensure that you and your team are notified of new comments instantly, enabling prompt responses and community engagement.
+In this section, you will learn how to set up a flow to send comments email to a specified email.  
+
+:::info Box title
+
+If you are self-hosting a Directus instance, you will need to set up the [email service configuration](https://docs.directus.io/self-hosted/config-options.html#email).
+
+:::
+### How to Implement
+Create a flow to trigger email upon creating a comment and select the `comment` collection. 
+Now add a **Read Data** operation attached to the Content Translations collection with the ID `{{$trigger.payload.content}}`.  This will return the comment that was just created. Now attach the **Send Email** operation. In the **To** section input the email addresses you want the notification to go to.
+**Subject**:  Comment for on article "{{get_article_comment.title}}"
+**Type**: Markdown
+**Body**: 
+
+    Hello, there is a comment for the article "{{get_article_comment.title}}"
+    > {{$trigger.payload.comment}}
+![email alert](email-alert.png)
```

Comment:
````suggestion
## 3. Configuring Alerts for New Comments

![comment alert](comment-alert.png)

As your blog grows, monitoring comments manually becomes impractical. Automated alerts ensure that you and your team are notified of new comments instantly, enabling prompt responses and community engagement.

In this section, you will learn how to set up an automation to send comments email to a specified email.  

This automation assumed that comments are either collected directly in your Directus project, or that they are synced/pushed via an integration.

:::info Email Setup

If you are self-hosting a Directus instance, you will need to set up the [email service configuration](https://docs.directus.io/self-hosted/config-options.html#email).

:::


Create a flow to trigger email upon creating a comment and select the `comment` collection. 

Now add a **Read Data** operation attached to the Content Translations collection with the ID <span v-pre>{{$trigger.payload.content}}`</span>. This will return the comment that was just created. Now attach the **Send Email** operation. In the **To** section input the email addresses you want the notification to go to.

```
**Subject**:  Comment for on article "{{get_article_comment.title}}"
**Type**: Markdown
**Body**: 
    Hello, there is a comment for the article "{{get_article_comment.title}}"
    > {{$trigger.payload.comment}}
```

![email alert](email-alert.png)
````